compiler/plugins/target/AMD-AIE/iree-amd-aie/Test/samples/matmul_pack_peel_air_e2e.mlir

```diff
@@ -18,14 +18,14 @@ func.func @matmul_i8_i32(%lhs: tensor<32x16xi8>, %rhs: tensor<16x32xi8>) -> tens
 
 // -----
 
-func.func @matmul_bf16(%lhs: tensor<16x32xbf16>, %rhs: tensor<32x16xbf16>) -> tensor<16x16xbf16>
```

Comment:
Why is this output data type change needed?

Reply:
We don't support matmul with output type bf16. The lowering of the vector.contract to aievec.matmul fails (there is a compilation error, I can't remember exactly what it is). Because this PR adds the lowering from vector to aievec, we only hit this error with this PR.

Reply:
I see, thanks!